tasks/generate-ca.yml

```diff
@@ -13,8 +13,7 @@
     - ca-config.json
 
 - name: Generate CA certificate on the first node of inventory group {{ etcd_cert_default_inventory_group_name }}
-  delegate_to: "{{ groups[etcd_cert_default_inventory_group_name][0] }}"
```

Comment:
Where do you remove `etcd_cert_default_inventory_group_name` variable from defaults?